frontend/src/component/releases/ReleaseManagement/ReleasePlanTemplateList.tsx

```diff
@@ -1,6 +1,7 @@
 import { Grid, styled } from '@mui/material';
 import { ReleasePlanTemplateCard } from './ReleasePlanTemplateCard/ReleasePlanTemplateCard';
 import type { IReleasePlanTemplate } from 'interfaces/releasePlans';
+import { ReleasesFeedback } from '../ReleasesFeedback/ReleasesFeedback';
```

Comment:
Can I suggest we move this component so we don't have to drop a folder. We have an [ADR](https://docs.getunleash.io/contributing/ADRs/front-end/preferred-folder-structure) for this!

Reply:
Yes, I get anxious when I see our folder structure!